lib/upfluence/error_logger/sentry.rb

```diff
@@ -128,6 +170,22 @@ def unit_name
       def unit_type
         unit_name&.split('@')&.first
       end
+
+      def set_error_level(event, error)
+        return :error if error.nil?
```

Comment:
here and L187, returning a symbol is not useful. 

could you remove the event argument, rename the method `compute_error_level` andchange the L78 like: `event.level = compute_error_level(exc)`

Also renaming the `error` error arg to `exc`